dataduct/pipeline/data_pipeline.py

```diff
@@ -2,13 +2,16 @@
 Base class for data pipeline instance
 """
 from collections import defaultdict
-
+from boto.datapipeline import regions
 from boto.datapipeline.layer1 import DataPipelineConnection
+from dataduct.config import Config
 
 from .pipeline_object import PipelineObject
 from .utils import list_pipeline_instances
 from ..utils.exceptions import ETLInputError
 
+config = Config()
+REGION = config.etl.get('REGION')
```

Comment:
`REGION = config.etl.get('REGION', None)` so that it doesn't break if the config is missing a region and it just defaults to whatever version boto uses.